shuup/admin/static_src/base/scss/variables.scss

```diff
@@ -1,10 +1,70 @@
-// Brand colors
+// ANTIQUATED Brand colors
 $primary: #07B0F2;
 $secondary: #041E40;
 $success: #27ae60;
 $warning: #e67e22;
 $danger: #e74c3c;
 $info: #228FB7;
+$text-color: #384850;
+
+// Precompile colors before adding to root as CSS variables
+$colors: (
+    primary: #07B0F2,
```

Comment:
shouldn't you use a variable here instead?

Reply:
Updated :)